src/main/java/nextstep/courses/domain/session/SessionImage.java

```diff
@@ -1,11 +1,29 @@
 package nextstep.courses.domain.session;
 
-import nextstep.file.ImageFile;
+import nextstep.file.domain.ImageFile;
 
 public class SessionImage {
 	ImageFile imageFile;
 
-	public SessionImage(int size, int width, int height, String type) {
+	public SessionImage(Long id) {
+		this.imageFile = new ImageFile(id);
```

Comment:
```suggestion
		this(new ImageFile(id));
```
부생성자가 주생성자를 호출해 구현할 것을 추천